llvm/lib/Target/AMDGPU/SIISelLowering.cpp

```diff
@@ -14797,16 +14875,19 @@ SDValue SITargetLowering::performClampCombine(SDNode *N,
   return SDValue(CSrc, 0);
 }
 
-
 SDValue SITargetLowering::PerformDAGCombine(SDNode *N,
                                             DAGCombinerInfo &DCI) const {
+  SelectionDAG &DAG = DCI.DAG;
```

Comment:
Not sure why this was moved here. It is not used in the added code anyway.

Reply:
I have already remove it.